### DISCUSSION TITLE
Rendre les videos responsives

### DISCUSSION THREAD

**Discussion by innermost47** [AmauriC/tarteaucitron.js#775] [Q&A]
Salut !

Déjà merci pour le taf, t'as assuré !
Dis moi, lorsque j'utilise tarteaucitron pour mes videos YouTube, je vois qu'il ne m'est pas possible de régler le height en %, juste en px, du coup je ne peux pas rendre mes vidéos responsives.
Comment faire please ?

**Comment by sanzefor** [comment]
Bonjour :)

Même question que @[innermost47](https://github.com/innermost47)

**Comment by AmauriC** [comment]
Hello : )

En ajoutant % ça devrait passer, il n'y a pas de `px` ajouté sur la taille.
